api/src/test/java/org/openmrs/validator/ObsValidatorTest.java

```diff
@@ -513,4 +515,20 @@ public void validate_shouldPassValidationIfValueTextIsLessThanTheMaximumLength()
 		Assert.assertFalse(errors.hasFieldErrors("obsDatetime"));
 		Assert.assertFalse(errors.hasFieldErrors("valueText"));
 	}
+
+	@Test
+	public void validate_shouldPassValidationForEqualObsPersonValueWithEncounterPatient() {
+		EncounterService es = Context.getEncounterService();
+		PersonService ps = Context.getPersonService();
+
+		Obs obs = new Obs();
+		obs.setPerson(ps.getPerson(2)); // Setting this person (2) as different from that of encounter person (7)
+		obs.setEncounter(es.getEncounter(2));
```

Comment:
Would be nice to also have an assertion here that the person objects are different just in case the test data changes

Reply:
@wluyima ok I will add an assertion :)